weekly-updates/weekly-update.2015-07-24.md

```diff
@@ -0,0 +1,14 @@
+# io.js and Node.js News
+This week we didn't have io.js releases, complete changelog from previous releases can be found [on GitHub](https://github.com/nodejs/io.js/blob/master/CHANGELOG.md).
```

Comment:
Nit: This is a comma splice. I would recommend changing to two separate sentences:

> This week, we did not have io.js releases. The complete changelog from previous releases can be found [on GitHub](https://github.com/nodejs/io.js/blob/master/CHANGELOG.md).